rcongui/src/components/PlayerInfo/PlayerInfo.js

```diff
@@ -0,0 +1,267 @@
+                PUNISH: 0,
+                KICK: 0
+            },
+            blacklist: {
+                is_blacklisted: false,
+                reason: "",
+                by: ""
+            },
+            flags: [],
+            watchlist: {},
+            steaminfo: {},
+            perma: false,
+            temp: false,
+            vip: false,
+            loaded: false
+        }
+    }
+
+    /**
+     * fetch bans that currently affect the steamId64
+     * @param steamId64
+     */
+    fetchPlayerBan(steamId64) {
+        get(`get_ban?steam_id_64=${steamId64}`)
+            .then((data) => {
+                const temp = data.result.find((ban, index) => {
+                    return ban.type === "temp";
+
+                })
+                const perma = data.result.find((ban, index) => {
+                    return ban.type === "perma";
+                })
+                if (temp !== undefined) {
+                    this.setState({temp: true})
+                }
+                if (perma !== undefined) {
+                    this.setState({perma: true})
+                }
+            })
+            .catch(handle_http_errors);
+    }
+
+    /**
+     * fetch Player data
+     * @param steamId64
+     */
+    fetchPlayer(steamId64) {
+        get(`player?steam_id_64=${steamId64}`)
+            .then((response) => showResponse(response, "get_user", false))
+            .then((data) => {
+                this.setState({
+                    created: data.result.created,
+                    names: data.result.names,
+                    sessions: data.result.sessions,
+                    sessions_count: data.result.sessions_count,
+                    total_playtime_seconds: data.result.total_playtime_seconds,
+                    current_playtime_seconds: data.result.current_playtime_seconds,
+                    received_actions: data.result.received_actions,
+                    penalty_count: data.result.penalty_count,
+                    blacklist: data.result.blacklist,
+                    flags: data.result.flags,
+                    watchlist: data.result.watchlist,
+                    steaminfo: data.result.steaminfo,
+                    loaded: true
+                })
+
+            })
+            .catch(handle_http_errors);
+    }
+
+    componentDidMount() {
+        const {steamId64} = this.props.match.params
+        if (steamId64 !== undefined) {
+            this.fetchPlayer(steamId64)
+            this.fetchPlayerBan(steamId64)
+        }
+    }
+
+    render() {
+        const {classes} = this.props
+        return (
+            <Grid container className={classes.root}>
+
+                {(this.state.loaded) ?
+                    <Grid item sm={12} className={classes.marginTop}>
+                        <Grid container>
+                            <Grid item sm={2}>
+                                <Grid container justify={"center"} alignContent={"center"} wrap={"wrap"}
```

Comment:
you don't need the curly braces when passing a string

Reply:
Just my ide that do that